openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -530,7 +530,8 @@ def _record_utm_registration_attribution(request, user):
             # We divide by 1000 here because the javascript timestamp generated is in milliseconds not seconds.
             # PYTHON: time.time()      => 1475590280.823698
             # JS: new Date().getTime() => 1475590280823
-            created_at_datetime = datetime.datetime.fromtimestamp(int(created_at_unixtime) / float(1000), tz=UTC)
+            created_at_datetime = datetime.datetime.fromtimestamp(
+                int(created_at_unixtime) / float(1000), tz=get_utc_timezone())
```

Comment:
```suggestion
            created_at_datetime = datetime.datetime.fromtimestamp(
                int(created_at_unixtime) / float(1000), tz=get_utc_timezone()
                )
```
Not sure how linters take this but something of this sort to make it more visible

Reply:
updated